EasyClangComplete.py

```diff
@@ -174,12 +176,14 @@ def on_post_save_async(self, view):
 
         """
         # disable on_activated_async when running tests
+        self.saving = True
```

Comment:
Not sure why I didn't apply mutex in the first place. I've removed all those bool flags and used mutex. As far as I can tell stability has improved.